pkg/cli/admin/release/extract.go

```diff
@@ -133,6 +142,9 @@ func NewExtract(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 	flags.StringVar(&o.CommandOperatingSystem, "command-os", o.CommandOperatingSystem, "Override which operating system command is extracted (mac, windows, linux) or can be specified with arch(linux/arm64, mac/amd64). You map specify '*' to extract all tool archives.")
 	flags.StringVar(&o.FileDir, "dir", o.FileDir, "The directory on disk that file:// images will be copied under.")
 
+	flags.BoolVar(&o.Included, "included", o.Included, "Extract credential request manifests only")
```

Comment:
Thanks :)  I've pushed 6a5869577 -> ae4ddeddc to shuffle doc-strings for the various filter parameters, because it's easier (at least for me) to understand how multiple exclusions fit together than it is to reason about how multiple inclusion properties fit together.